q2_sample_classifier/__init__.py

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env python
-
 # ----------------------------------------------------------------------------
 # Copyright (c) 2017--, q2-sample-classifier development team.
```

Comment:
Besides the date issue @ebolyen mentioned, should the Copyright notice name the `QIIME 2 development team`, like in other projects within @qiime2?